src/views/questionView.js

```diff
@@ -12,15 +12,20 @@ export const createQuestionElement = (question) => {
 
   // I use String.raw just to get fancy colors for the HTML in VS Code.
   element.innerHTML = String.raw`
-    <h1>${question}</h1>
+  <h1>${question}</h1>
 
-    <ul id="${ANSWERS_LIST_ID}">
+    <ul id="${ANSWERS_LIST_ID}" class='list_of_answers'>
```

Comment:
This class is not used anywhere so don't think you need to add it

Reply:
done